test/extended/cluster/utils.go

```diff
@@ -519,3 +519,39 @@ func SetNamespaceLabels(c kclientset.Interface, name string, labels map[string]s
 	ns.Labels = labels
 	return c.CoreV1().Namespaces().Update(ns)
 }
+
+func ProjectExists(oc *exutil.CLI, name string) (bool, error) {
+	projectList, err := oc.AdminProjectClient().Project().Projects().List(metav1.ListOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	for _, p := range projectList.Items {
+		if name == p.Name {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+func DeleteProject(oc *exutil.CLI, name string, sync bool, interval, timeout time.Duration) error {
+	e2e.Logf("Deleting project %v ......", name)
```

Comment:
Three dots (an ellipsis) is more than enough.

Reply:
Done.